homeassistant/components/numato/__init__.py

```diff
@@ -0,0 +1,211 @@
+"""
+Support for controlling GPIO pins of a Numato Labs USB GPIO expander.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/integrations/numato
```

Comment:
We don't add the url to the docstring anymore. Just keep the very first line.

Reply:
Done.